client/platform/web-girder/views/Summary.vue

```diff
@@ -1,13 +1,109 @@
 <script lang="ts">
-import { defineComponent } from '@vue/composition-api';
+import {
+  defineComponent, ref, computed,
+} from '@vue/composition-api';
+import { itemsPerPageOptions } from 'dive-common/constants';
+import { generateColors } from 'vue-media-annotator/use/useStyling';
+import * as d3 from 'd3';
+import { clientSettings } from 'dive-common/store/settings';
+import { getLabels } from '../api';
 
+
+type Label = {
+  _id: string;
+  count: number;
+  datasets: {
+    [x: string]:
+    {
+      id: string;
+      name: string;
+      color: string;
+      };
+  };
+};
 export default defineComponent({
   setup() {
-    console.log('whatever');
+    const summaryList = ref();
+    const expanded = ref();
+    const headers = [
+      { text: 'Type', value: '_id' },
+      { text: 'Number of Datasets', value: 'datasets.length' },
+      { text: 'Number of Tracks', value: 'count' },
+      { text: '', value: 'data-table-expand' },
+    ];
+
+    const labelColors = generateColors(10);
+    const ordinalColorMapper = d3.scaleOrdinal<string>().range(labelColors);
+
+    const statsLabelStyling = computed(() => ({
+      color: (label: string) => ordinalColorMapper(label),
+    }));
+
+
+    const updateList = async () => {
+      const response = await getLabels();
+
+      summaryList.value = response.data;
+
+      summaryList.value.forEach((item: Label) => {
+        Object.keys(item.datasets).forEach((key) => {
+          // eslint-disable-next-line no-param-reassign
+          item.datasets[key].color = statsLabelStyling.value.color((item.datasets[key].name));
+        });
+      });
+    };
+    updateList();
+
+
+    return {
+      expanded,
+      summaryList,
+      headers,
+      clientSettings,
+      itemsPerPageOptions,
+    };
   },
 });
 </script>
 
 <template>
-  <p>Summary placeholder</p>
+  <v-data-table
+    :headers="headers"
+    :items="summaryList"
+    :expanded.sync="expanded"
+    :items-per-page.sync="clientSettings.rowsPerPage"
+    :footer-props="{ itemsPerPageOptions }"
+    item-key="_id"
+    show-expand
+  >
+    <template
+      v-slot:expanded-item="{ headers, item }"
+    >
+      <td :colspan="headers.length">
+        <v-chip
+          v-for="dataset in item.datasets"
+          :key="dataset.id"
```

Comment:
@subdavis This (and subsequent instances) throws an error but I am unsure how to assign the type inside the template slot.

Reply:
Out of curiosity, what error do you get?

Reply:
It's a type error from Vetur `Object is of type 'unknown'`. Still compiles but I would prefer to fix if possible.

Reply:
You want the `key` to be unique within the component instance.

Because dataset.id appears more than once (in different rows) you're getting an error because vue can't disambiguate.

You should construct a compound key that includes both `_id` and `dataset.id`.

Reply:
I attempted to fix but still having issues. I _think_ its because the item is inherently type `any` in the slot:
https://vuetifyjs.com/en/api/v-data-table/#slots-expanded-item

Reply:
@subdavis